### ISSUE TITLE
Request spec example with parsing JSON with symbolized keys

### ISSUE THREAD

**Issue by eliotsykes** [eliotsykes/rspec-rails-examples#9]
Add request spec example that parses JSON and allows symbol access to resulting Hash (a little more elegant than default String key access to hash)

```ruby
JSON.parse(response.body, symbolize_names: true)
```

**Comment by eliotsykes** [comment]
Create `spec/support/json_helper.rb` to provide `json` helper method.

```ruby
module JsonHelper
   def json
     @json ||= JSON.parse(response.body, symbolize_names: true)
   end
end

RSpec.configure do |config|
  config.include JsonHelper, type: :request
end
```

**Comment by eliotsykes** [comment]
WIP: `JsonHelper` making an appearance in #96

**Comment by eliotsykes** [comment]
Closed by #96
